src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -153,44 +202,13 @@ export function useRegistrationValidation(): StepValidation {
           'Valid certificate must be present if you are registering Satellite.',
       });
     }
-    if (registrationCommand === '' || !registrationCommand) {
-      Object.assign(errors, {
-        command: 'No registration command for Satellite registration',
-      });
-    }
-    try {
-      const match = registrationCommand?.match(
-        /Bearer\s+([\w-]+\.[\w-]+\.[\w-]+)/
-      );
-      if (!match) {
-        Object.assign(errors, { command: 'Invalid or missing token' });
-      } else {
-        const token = match[1];
-        const decoded = jwtDecode(token);
-        if (decoded.exp) {
-          const currentTimeSeconds = Date.now() / 1000;
-          const dayInSeconds = 86400;
-          if (decoded.exp < currentTimeSeconds + dayInSeconds) {
-            const expirationDate = new Date(decoded.exp * 1000);
-            Object.assign(errors, {
-              expired:
-                'The token is already expired or will expire by next day. Expiration date: ' +
-                expirationDate,
-            });
-            return {
-              errors: errors,
-              disabledNext: caCertificate === undefined,
-            };
-          }
-        }
-      }
-    } catch {
-      Object.assign(errors, { command: 'Invalid or missing token' });
-    }
+    const tokenErrors = validateSatelliteToken(registrationCommand);
+    Object.assign(errors, tokenErrors);
+
     return {
       errors: errors,
       disabledNext:
-        Object.keys(errors).length > 0 || caCertificate === undefined,
+        Object.keys(errors).filter((key) => key !== 'expired').length > 0,
```

Comment:
nitpick: `Object.keys(errors).some(key => key !== 'expired')` is shorter and faster.

Reply:
Thanks for the review. I think that the current validation is much more readable. ;)